core/src/components/cat-scrollable/cat-scrollable.tsx

```diff
@@ -0,0 +1,31 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'cat-scrollable',
+  styleUrl: 'cat-scrollable.css',
+  shadow: true,
+})
+export class CatScrollable {
+
+  /** Flags to enable/disable scroll shadows. */
+  @Prop()
+  shadow: { x?: boolean; y?: boolean; } = { x: true, y: true };
```

Comment:
same here

Reply:
as far as I read in best practices, it is discouraged to use complex objects in properties. So I suggest to go with individual props like shadowX and shadowY